### ISSUE TITLE
[package request] Radium, the next-generation tracker/music editor

### ISSUE THREAD

**Issue by userabc123xyz** [void-linux/void-packages#25662]
Radium is a unique tracker-like music editor with a new type of interface
The official page: http://users.notam02.no/~kjetism/radium/

**Comment by ScrelliCopter** [comment]
I took a brief look and it appears that the current stable version requires the Steinberg VST 2.4 SDK to build, which is proprietary and no longer even possible to obtain legally. I doubt this could be packaged unless there's a way to disable VST 2.4 support.

**Comment by uzr123x** [comment]
@ScrelliCopter that's not true, VST 2.x SDK comes with VST 3 SDK archive which you can freely download from https://new.steinberg.net/developers/

**Comment by ScrelliCopter** [comment]
Looks like you're right and they must've added it back (circa 2018 it was removed from the SDK distribution entirely), however I think you'll find the license hasn't changed and is still more or less proprietary. VST2.4 support may still need to be disabled for inclusion into void but I won't pretend to speak on behalf of the core team :)

**Comment by uzr123x** [comment]
@ScrelliCopter why not put it into Void's nonfree repo then?

**Comment by ScrelliCopter** [comment]
Would probably be okay
